CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2023 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+cmake_minimum_required(VERSION 3.13)
+
+if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.15)
+  # MSVC runtime library flags are selected by the CMAKE_MSVC_RUNTIME_LIBRARY abstraction.
+  cmake_policy(SET CMP0091 NEW)
+  # MSVC warning flags are not in CMAKE_<LANG>_FLAGS by default.
+  cmake_policy(SET CMP0092 NEW)
+endif()
+if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.20)
+  # MSVC RTTI flag /GR is not added to CMAKE_CXX_FLAGS by default.
+  cmake_policy(SET CMP0117 NEW)
+endif()
```

Comment:
nit: CMake 3.13 is from Nov 2018. Is there any reason to support that? Why not just `cmake_minimum_required(VERSION 3.20)` and remove those `if`s? (I guess some laggish linux distro still ships with 3.13?)

Reply:
> I guess some laggish linux distro still ships with 3.13?

Debian 10 Buster.

Reply:
> nit: CMake 3.13 is from Nov 2018.

The [draft](https://github.com/bitcoin/bitcoin/pull/25797) supports even CMake 3.10 which is shipped with Ubuntu Bionic :)

Reply:
Related: today's IRC meeting [log](https://www.erisian.com.au/bitcoin-core-dev/log-2023-02-09.html#l-270).